evm/verify-contract.js

```diff
@@ -320,6 +330,46 @@ async function processCommand(config, chain, options) {
             break;
         }
 
+        case 'AxelarAmplifierGateway': {
+            let args = options.args;
+
+            if (args) {
+                args = JSON.parse(args);
+                options = { ...options, ...args };
+            }
```

Comment:
avoid merging the two. read from `args` directly if that's where a value is defined. This becomes quite confusing for readability (if I see options.xyz later, I expect an `--xyz` flag to be defined)